compiler-core/src/erlang.rs

```diff
@@ -223,7 +223,7 @@ fn module_document<'a>(
     ));
 
     Ok(header
-        .append("-compile(no_auto_import).")
+        .append("-compile([no_auto_import, nowarn_unused_vars]).")
```

Comment:
Might this result in us not detecting bugs?
Could this be hidden behind a flag? Would that make sense?

Reply:
Gleam detects these already so the Erlang warning is a duplicate